src/item.cpp

```diff
@@ -9207,6 +9207,27 @@ const itype_variant_data &item::itype_variant() const
     return *_itype_variant;
 }
 
+std::string item::pick_random_variant()
```

Comment:
You could probably use this instead:
https://github.com/CleverRaven/Cataclysm-DDA/blob/e0694d002ca10f5329132a8bd9d91ed0aca1d5bf/src/item.cpp#L9170-L9184

Reply:
Good call again. Both of your remaining points should be resolved now. Going to give it some testing (and time for you to respond in case I really screwed the pooch) and will be undrafting